ui/pages/Address.tsx

```diff
@@ -346,7 +346,14 @@ const AddressPageContent = () => {
         />
       ) }
       <AddressEntity
-        address={{ ...addressQuery.data, hash: checkSummedHash, name: '', ens_domain_name: '', implementations: null }}
+        address={{
+          ...addressQuery.data,
+          hash: checkSummedHash,
+          filecoin: addressQuery.data?.filecoin,
```

Comment:
I think this change is unnecessary since the whole address data is passed above:
```
...addressQuery.data,
```